src/uv_tcp.h

```diff
@@ -13,7 +13,8 @@ struct UvTcp
     struct uv_loop_s *loop;              /* Event loop */
     raft_id id;                          /* ID of this raft server */
     const char *address;                 /* Address of this raft server */
-    struct uv_tcp_s listener;            /* Listening TCP socket handle */
+    size_t num_listeners;                /* The number of listener sockets (multi IPs) */
```

Comment:
Nit: the type/naming convention we have elsewhere for this kind of counters is `unsigned n_listeners` (so `n_<name-of-array>`).

We're using `unsigned` because we're not really holding the size of a memory object and `unsigned` is big enough for our purposes (it's guaranteed to be at least 16 bits, so up to 65535).

Reply:
Renamed the member to n_listeners and changed the type to unsigned.